bouncy/release-bionic-arm64-build.yaml

```diff
@@ -0,0 +1,90 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_label: buildagent_arm64 || bouncy_binarydeb_ubv8
+jenkins_binary_job_priority: 82
+jenkins_binary_job_timeout: 720
+jenkins_source_job_priority: 78
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - mikael+build.ros2.org@openrobotics.org
```

Comment:
Please also create and use a google group (as in ROS 1).

Same below for amd as well as the index.

Reply:
Fair point, I copied from ardent without looking further. I'll create a google group as we do for the ROS farm :+1:

Reply:
group has been created: https://groups.google.com/forum/#!forum/ros2-buildfarm-bouncy. Need to invite the buildfarm user once we have emails configured on the farm